events/2016/1-nodeschool.md

```diff
@@ -1,7 +1,8 @@
 ---
 name: Nodeschool
+slug: nodeschool
```

Comment:
Would it make more sense to generate the slug from the filename? That would help keep things consistent and reduce the complexity of the metadata section.